tests/components/mqtt/test_cover.py

```diff
@@ -2047,8 +2047,9 @@ async def test_tilt_position_altered_range(hass, mqtt_mock_entry_with_yaml_confi
     )
 
 
-async def test_find_percentage_in_range_defaults(hass):
+async def test_find_percentage_in_range_defaults(hass, mqtt_mock_entry_no_yaml_config):
     """Test find percentage in range with default range."""
+    await mqtt_mock_entry_no_yaml_config()
```

Comment:
Are these changes for another PR?

Reply:
No, this because the current tests do not load the config entry.

Reply:
But I agree these changes should not be needed.